src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -0,0 +1,584 @@
+                                        for (NewGTFSError error : primaryKeyErrors) {
+                                            if (error.errorType.equals(NewGTFSErrorType.DUPLICATE_ID)) {
+                                                String key = String.join(":", table.name, idScope, val);
+                                                // Modify route_id and ensure that referencing trips have route_id updated.
+                                                val = String.join(":", idScope, val);
+                                                mergeFeedsResult.remappedIds.put(key, val);
+                                            }
+                                        }
+                                    }
+                                } else {
+                                    // Key field has defaulted to the standard primary key field (stop_id or route_id),
+                                    // which makes the check much simpler (just skip the duplicate record).
+                                    for (NewGTFSError error : idErrors) {
+                                        if (error.errorType.equals(NewGTFSErrorType.DUPLICATE_ID)) {
+                                            skipRecord = true;
+                                        }
+                                    }
+                                }
+                                break;
+                            case "transfers":
+                                // For any other table, skip any duplicate record.
+                                for (NewGTFSError error : idErrors) {
+                                    if (error.errorType.equals(NewGTFSErrorType.DUPLICATE_ID)) {
+                                        skipRecord = true;
+                                    }
+                                }
+                                break;
+                            default:
+                                // For any other table, skip any duplicate record.
+                                for (NewGTFSError error : idErrors) {
+                                    if (error.errorType.equals(NewGTFSErrorType.DUPLICATE_ID)) {
+                                        skipRecord = true;
+                                    }
+                                }
+                                break;
+                        }
+                        String valueToWrite = val;
+                        if (field.isForeignReference()) {
+                            // If the field is a foreign reference, check to see whether the reference has been
+                            // remapped due to a conflicting ID from another feed (e.g., calendar#service_id).
+                            String key = String.join(":", field.referenceTable.name, idScope, val);
+                            if (mergeFeedsResult.remappedIds.containsKey(key)) {
+                                mergeFeedsResult.remappedReferences++;
```

Comment:
Should this if block be moved after the skippedIds check? Seems like that way this remappedReferences counter will not be updated in the event that the id was skipped.